src/contrib/cluster/Akka.DistributedData/Internal/Internal.cs

```diff
@@ -878,7 +888,7 @@ private sealed class NoDelta : IDeltaReplicatedData<IReplicatedData, IReplicated
         /// </summary>
         public static readonly IReplicatedDelta NoDeltaPlaceholder = NoDelta.Instance;
 
-        public readonly UniqueAddress FromNode;
+        public UniqueAddress FromNode { get; }
```

Comment:
Why is this a property but everything else is a field?

Reply:
Because of the new interface `ISendingSystemUid`

Reply:
Why not make everything properties then? Just for consistency.

Reply:
Mainly to keep the change to the API and possible impact from code change as minimum as possible, should I change them all to properties?

Reply:
Yeah I would - if we're going to break binary compatibility (actually this may not be binary-incompatible since the return types, member names, are the same) we might as well clean it up. We use properties everywhere inside most message types.

Reply:
Alright, I'll do a code clean-up then

Reply:
Done